### PR TITLE
fix institute naming

### PR DESCRIPTION
**Pull request by tobyhodges**
The content was inconsistent, referring to the NCBI in general and the ENA specifically. I've changed it to refer to NCBI and EMBL-EBI (the institutes), but happy to adjust to refer specificallty to the SRA and ENA, if you prefer that?

### CODE REVIEW

**Comment by hoytpr** [comment]
I like this question @tobyhodges! Being accurate and consistent is super important. But this issue might have been considered because the EMBL-EBI landing page does not seem to have a direct link to the ENA, while the NCBI page does (albeit in small letters at the bottom). From EMBL-EBI you have to click on Data Resources, and then click again on "A-Z" to find ENA. My bias is to put both links to both sites *e.g.* 
"the [National Center for Biotechnology Information  (NCBI)](https://www.ncbi.nlm.nih.gov) has the [Sequence Read Archive (SRA)](https://trace.ncbi.nlm.nih.gov/Traces/sra/) and [European Nucleotide Archive (EMBL-EBI)](https://www.ebi.ac.uk/) has their [Nucleic Acid Archive](https://www.ebi.ac.uk/ena)." Naturally I'm going to ask @tracykteal for comment. 

EDIT: Notice this is mixed in with issue #98 but looks manageable.

**Comment by tobyhodges** [comment]
Thanks for the feedback, Pete. I'll be happy to update the PR accordingly after we've heard from Tracy.
@tracykteal

File: _episodes/03-ncbi-sra.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ keypoints:
...
 
-There are many repositories for public data. Some model organisms or fields have specific databases, and there are ones for particular types of data. Two of the most comprehensive are the [National Center for Biotechnology Information  (NCBI)](https://www.ncbi.nlm.nih.gov) and [European Nucleotide Archive (EMBL-EBI)](https://www.ebi.ac.uk/ena). In this lesson we're working with the NCBI database, but the general process is the same for any database.
+There are many repositories for public data. Some model organisms or fields have specific databases, and there are ones for particular types of data. Two of the most comprehensive are provided by the [National Center for Biotechnology Information (NCBI)](https://www.ncbi.nlm.nih.gov) and the [European Bioinformatics Institute (EMBL-EBI)](https://www.ebi.ac.uk/). In this lesson we're working with the NCBI database, but the general process is the same for any database.
```
Comment (tracykteal) [inline _episodes/03-ncbi-sra.md:17]: To both acknowledge the institute and point to the resources, how about "the [European Bioinformatics Institute (EMBL-EBI)](https://www.ebi.ac.uk/) through the [European Nucleotide Archive](https://www.ebi.ac.uk/ena)."

Comment (hoytpr) [reply _episodes/03-ncbi-sra.md:17]: Sounds great to me. @tobyhodges  if you can make the change, we should be able to merge. Thanks!

Comment (tobyhodges) [reply _episodes/03-ncbi-sra.md:17]: :+1:

Comment (tracykteal) [reply _episodes/03-ncbi-sra.md:17]: Thanks @tobyhodges!

**Pull request merged**